coderd/database/migrations/000255_workspace_agent_script_timings.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE workspace_agent_script_timings
+(
+    job_id       uuid not null references provisioner_jobs (id) on delete cascade,
```

Comment:
This relationship is probably unwise (purely from the data modelling perspective - I understand why you did this, though). I think we should instead link to the workspace agent, and then craft a query which will determine the agent_scripts associated to the workspace_resources associated to the provisioner_job - to locate the timings. I think that would be most congruent with our current model, albeit a little less efficient to query.

WDYT?

Reply:
Yeah it definitely does feel odd as is. I'm happy to make this change if you think it makes more sense that way.